src/Analysis/SharedUtils/TeamProject/Utils/CheckForTeamProjectApplication.js

```diff
@@ -0,0 +1,13 @@
+const TeamProjectApplications = ['OHDSI Atlas', 'GWASUIApp', 'GWASResults'];
```

Comment:
It would be nice to not have to rely on the names like this, but instead have an attribute that can function as a switch for this feature (in `gitops.json` config file)

Reply:
Will follow up with you regarding this.